crates/dbsp/src/trace/layers/erased/erased_key_leaf/builders.rs

```diff
@@ -0,0 +1,233 @@
+use crate::{
```

Comment:
this file is very abstract, but also simple.
I can't really see what the goal is